background/handle-auth.js

```diff
@@ -7,178 +7,216 @@ const promisify =
   (...args) =>
     new Promise((resolve) => callbackFn(...args, resolve));
 
+const CHROME_DEFAULT = "0";
+const FIFEFOX_DEFAULT = "firefox-default";
 async function getDefaultStoreId() {
```

Comment:
There's nothing wrong with defining constants inside the function they're used in.

Reply:
`Fifefox`? Do you mean Firefox?

Reply:
Whoops, it was my tipo after all.